### PR TITLE
Firefox U2F support

### PR DESCRIPTION
**Pull request by Airtune**
* webpack TransportU2F
* implement U2F block signer

### CODE REVIEW

**Comment by Airtune** [comment]
~~I want to look through this one more time before we merge it in~~

File: index.js [hunk 1]
```diff
@@ -157,7 +189,11 @@
       }
     };
     return signer;
-  };
+  }
+
+  const onUsbReady = async (callback) => {
```
Comment (Airtune) [inline index.js:194]: Does the CLI need a onUsbReady check before calling the callback?

Comment (Airtune) [inline index.js:194]: Does the node version, i.e. non-browser version, need a check in onUsbReady before calling the callback?

This is one thing I haven't tested yet.

**Comment by Airtune** [comment]
> I want to look through this one more time before we merge it in

I've done the looking through I wanted to now

File: web/bananojs-hw.js [hunk 2]
```diff
@@ -83,13 +226,13 @@ window.bananocoin.bananojsHw.getLedgerAccountSigner = async (accountIx) => {
...
         hwBlockData.recipient = window.bananocoinBananojs.getBananoAccount(
-            blockData.link,
+          blockData.link,
         );
 
         const cacheBlockData = {};
         const cacheBlocks = await bananodeApi.getBlocks(
-            [blockData.previous],
-            true,
+          [blockData.previous],
```
Comment (dev-ptera) [inline web/bananojs-hw.js:235]: One day I'll add prettier to this package to prevent indentation all over the place.

File: index.js (hunk 1 above)
Comment (dev-ptera) [reply index.js:194]: I've also never tested this package outside of a browser; does this package even support non-browser HW connectivity?

**Comment by Airtune** [comment]
U2F support is dropped by Mozilla Firefox

**Pull request closed without merging**